strawberry/sanic/views.py

```diff
@@ -79,7 +79,7 @@ def __init__(
 
             self.json_encoder = json.JSONEncoder
 
-    def get_root_value(self):
+    def get_root_value(self) -> None:
```

Comment:
I think we want Any or object here since we want subclasses to override this method

Reply:
then I guess `Any` would be more adequate.

I don't really understand the use of `object` in typing, since it is the base class for everything else, so might as well use `Any` IMO